Sources/Public/Animation/LottieView.swift

```diff
@@ -106,9 +106,10 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     _ loadAnimation: @escaping () async throws -> LottieAnimationSource?,
     @ViewBuilder placeholder: @escaping () -> Placeholder)
   {
+
     self.loadAnimation = loadAnimation
+    self.localAnimation = nil
```

Comment:
applying the change the linter is complaining about

```suggestion
    localAnimation = nil
    self.loadAnimation = loadAnimation
```